src/Cool/DidYouKnow/didYouKnowModal.cpp

```diff
@@ -0,0 +1,98 @@
+#include "didYouKnowModal.hpp"
+#include <imgui.h>
+#include <vcruntime.h>
+#include <string>
+
+void DidYouKnowModal::open()
+{
+    auto difference = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now()) - std::chrono::system_clock::to_time_t(_current_timestamp);
+
+    size_t time_to_wait = 1 * 60 * 60; // 1 hour
```

Comment:
You can use chrono and write this as 1h. https://learn.microsoft.com/en-us/cpp/standard-library/chrono-literals?view=msvc-170#example
Also, you don't need to create a variable anymore since 1h is already pretty clear.